src/Core/Repositories/IOrganizationUserRepository.cs

```diff
@@ -37,5 +37,6 @@ public interface IOrganizationUserRepository : IRepository<OrganizationUser, Gui
         Task DeleteManyAsync(IEnumerable<Guid> userIds);
         Task<OrganizationUser> GetByOrganizationEmailAsync(Guid organizationId, string email);
         Task<IEnumerable<OrganizationUserPublicKey>> GetManyPublicKeysByOrganizationUserAsync(Guid organizationId, IEnumerable<Guid> Ids);
+        Task<IEnumerable<OrganizationUserUserDetails>> GetManyByMinimumRoleAsync(Guid organizationId, OrganizationUserType role);
```

Comment:
You forgot to change the `OrganizationUserType` parameter to match what you have in the implementation: `baseRole`

Reply:
Changed to minRole everywhere to match the original suggestion & new method name.